cfme/tests/configure/test_log_depot_operation.py

```diff
@@ -29,10 +28,9 @@ def __init__(self, protocol, credentials, access_dir=None, path=None):
         self.protocol = protocol
         self._param_name = self.protocol
         self.credentials = credentials
-        self.access_dir = access_dir
+        self.access_dir = access_dir if access_dir else ""
```

Comment:
**Optional**
I think it would be cleaner to use `or` here, and the behavior should be what you want. If access_dir is None (false-ish) it'll set an empty string. If access_dir is set (trueish) it'll use it, unless its set to `False`.

```python
self.access_dir = access_dir or ""
```